### ISSUE TITLE
Fleet as a standalone app

### ISSUE THREAD

**Issue by jezzdk** [aschmelyun/fleet#15]
Okay so it has bothered me a bit, that I need to install Fleet in all the projects that I want to add to Fleet. It's a bit redundant and also cumbersome when Fleet needs to be updated in potentially a lot of places.

So I took the liberty to create Fleet as a stand-alone app using [Laravel Zero](https://laravel-zero.com/). Basically it means that it's installed once, using `composer require global`, instead of locally in every single project. I've tested it locally and it works like a charm.

I could've made this as a pullrequest, but since it's a pretty big change I took another route. What I did, is that I created a new Laravel Zero app, and then I took all your business logic and copied it over. I only had to change a little bit since the stand-alone app doesn't have access to the project's local packages. For instance, `base_path()` would refer to the base path of the global composer package and not the project. 

I have published it to my own repo at the moment, but actually I want you to have it. If you want it :) It hasn't been published to Packagist yet, so feel free to clone it into your own account or whatever. But I'd like to keep myself included as an author if that's okay.

**Comment by jezzdk** [comment]
Oh right, the repo is here: https://github.com/jezzdk/fleet-cli

**Comment by jezzdk** [comment]
@aschmelyun Whattaya think? :)

**Comment by joseph-d** [comment]
I agree that it definitely seems a bit redundant / cumbersome to install fleet in every project. But beyond that, it's also a bit of a shame (from my perspective at least) to have to run any composer or php scripts on my host machine _at all_ because I can otherwise get Laravel installed and running using sail without having to "pollute" my host machine with anything.

Perhaps an even better approach to this would be a shell script, similar to the Laravel one, which pulls down the traefik image then does the composer install of this type of Laravel Zero-ified package of fleet (plus sail) using the laravelsail/php composer image and then maybe there could also be some sort of Laravel console command thingy to add new sites?

Ok, I admit there are a few gaps in my knowledge/explanation but if anyone can offer any guidance  then I'd be happy to give it a go.
